City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3322: Updated site configuration based on latest code

How to test:

1. Checkout this branch on this instance. If you don't have the instance set up yet you need to install the site with this branch `make new`
2. After install or if you already have the site installed run `make drush-cim && make drush-cr`
3. Now go to add content and make sure that the accordion for example has title and description fields or check the footer that there is the correct block "Helsingin kaupunki". Also make sure that the cookie page has all categories and translations.
